PowerEditor/installer/nsisInclude/gobalDef.nsh

```diff
@@ -34,6 +34,12 @@
 !define VERSION_MAJOR 7
 !define VERSION_MINOR 0
 
+!define CompanyName		"Don HO don.h@free.fr"
+!define Description		"Notepad++ : a free (GNU) source code editor"
+!define Version 		"7.0.0.0"
```

Comment:
@MAPJe71 Initially, I though to utilizing existing macros, but when I saw the history of these macros, I could not use these without writing conditions as the value. Consider V6.9.2, then major is 6, minor is 92. So break 92 as 9.2 and append .0 (6.9.2.0) while in V7 major is 7, minor is 0, so append .0.0 (so 7.0.0.0).